.ci/windows-steps.yaml

```diff
@@ -62,6 +62,7 @@ steps:
     -DBOOST_INCLUDEDIR=$(Agent.ToolsDirectory)\boost.1.60.0.0\lib\native\include `
     -DBOOST_LIBRARYDIR=$(Agent.ToolsDirectory)\boost_libs `
     -DCEREAL_INCLUDE_DIR=$(Agent.ToolsDirectory)\unofficial-flayan-cereal.1.2.2\build\native\include `
+    -DENSMALLEN_INCLUDE_DIR=$(Agent.ToolsDirectory)\ensmallen.2.16.2\include`
```

Comment:
I think this will go out of date as the ensmallen package changes version, unless we hardcode the version earlier or detect the version here or something.

Reply:
Agreed, we need to find a way to encode the version. In the meanwhile, let us see if we were able to install ensmallen at least.